### PR TITLE
feat: Create Changelog.yml ✨

### PR DESCRIPTION
**Pull request by Nishitbaria**
## Related Issue

Closes: #600 

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

### CODE REVIEW

**Comment by Nishitbaria** [comment]
### **Hey @anmode Please Review This pr**

File: .github/workflows/Changelog.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (anmode) [inline .github/workflows/Changelog.yml:18]: from where it comes?

File: .github/workflows/Changelog.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+      - name: Create Release
+        uses: actions/create-release@v1
+        if: ${{ steps.changelog.outputs.skipped == 'false' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.changelog.outputs.tag }}
+          release_name: ${{ steps.changelog.outputs.tag }}
+          body: ${{ steps.changelog.outputs.clean_changelog }}
```
Comment (anmode) [inline .github/workflows/Changelog.yml:29]: where these values passed on?

**Review by anmode — COMMENTED** [review]
need some ans!

File: .github/workflows/Changelog.yml (hunk 2 above)
Comment (Nishitbaria) [reply .github/workflows/Changelog.yml:29]: It is passing the generated changelog and the tag information to the TriPSs/conventional-changelog-action@v3 action, which might use this information to create a GitHub release, attach the changelog to a release, or perform other tasks related to versioning and release management. The specific functionality depends on how the TriPSs/conventional-changelog-action@v3 action is implemented.

**Pull request merged**